iodata/formats/molekel.py

```diff
@@ -112,7 +126,8 @@ def _load_helper_coeffs(lit: LineIterator, nbasis: int) -> Tuple[np.ndarray, np.
             ncol = len(words)
             assert ncol > 0
             for word in words:
-                assert word == 'a1g'
+                # assert word == 'a1g'
```

Comment:
Feel free to remove this commented line. This assert should never have been here in the first place.

Reply:
Removed. Thanks.